cqf-fhir-cr/src/test/java/org/opencds/cqf/fhir/cr/measure/r4/utils/TestDataGenerator.java

```diff
@@ -63,12 +64,17 @@ public void createPatient(String patientId, String practitionerId, String organi
         // Patient Creation
         Patient patient = new Patient();
         patient.setId(patientId);
+        Calendar calendar = Calendar.getInstance();
```

Comment:
I strong advise against using the ancient Calendar class.

I would do something like this:

```java
import java.time.ZoneId;

LocalDateTime localDateTime;
...
localDateTime =LocalDateTime.

of(1985,Month.JUNE, 16,14,30,0);
...
localDateTime =LocalDateTime.

of(1988,Month.JANUARY, 1!,14,30,0);
...
var date = Date.from(localDateTime.atZone(ZoneId.systemDefault()).toInstant())
```